packages/caver-core-method/src/index.js

```diff
@@ -348,15 +347,17 @@ const buildSendRequestFunc = (defer, sendSignedTx, sendTxCallback) => (payload,
                 // If wallet was not found in caver-js wallet, then it has to use wallet in Node.
                 // Signing to transaction using wallet in Node supports only LEGACY transaction, so if transaction is not LEGACY, return error.
                 if (tx.feePayer !== undefined || (tx.type !== undefined && tx.type !== 'LEGACY')) {
-                    error = new Error('Only Legacy transactions can be signed on a Klaytn node!')
+                    error = new Error(
+                        `Failed to send transaction: Only Legacy transactions can be signed on a Klaytn node. Please add an account(${addressToUse}) with caver.klay.accounts.wallet.add, or use a 'LEGACY' transaction.`
```

Comment:
ERROR: no private key found in the caver-js wallet. Trying to use the Klaytn node's wallet, but it only supports legacy transactions. Please add private key of ${addressToUse} to the caver-js wallet.

Please change the error statements like above.

Reply:
Error message is edited :) Thank you!